Description: Fix hurd build by disabling zfs code.
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Origin: upstream, http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/4237
Forwarded: not-needed
Applied-Upstream: http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/4237
Last-Update: 2012-06-08

--- a/grub-core/kern/emu/getroot.c	2012-04-21 14:49:56 +0000
+++ b/grub-core/kern/emu/getroot.c	2012-04-22 19:02:55 +0000
@@ -241,6 +241,8 @@ 
 
 #endif /* __linux__ */
 
+#if !defined (__GNU__)
+
 static char *
 find_device_from_pool (const char *poolname)
 {
@@ -322,6 +323,8 @@ 
   return device;
 }
 
+#endif
+
 #ifdef __MINGW32__
 
 char *
@@ -331,6 +331,8 @@ 
   return 0;
 }
 
+#elif defined (__GNU__)
+
 #elif ! defined(__CYGWIN__)
 
 char *

